src/test/unittest/unittest.sh

```diff
@@ -1252,11 +1254,24 @@ function require_dev_dax_node() {
 	DEVDAX_TO_LOCK=1
 }
 
+#
+# require_ndctl_enable -- check NDCTL_ENABLE value and skip test if set to 'n'
+#
+function require_ndctl_enable() {
```

Comment:
there's already a function that checks if pmdk was actually built with ndctl: `is_ndctl_enabled()`.